operations/common-operation.go

```diff
@@ -91,6 +107,13 @@ func CreateClusterAdmin(context *ClientContext) (sarama.ClusterAdmin, error) {
 		config.Net.TLS.Config = tlsConfig
 	}
 
+	if context.Sasl.Enabled {
+		output.Infof("SASL is enabled.")
```

Comment:
the log should be printed with `output.Debugf()` so that it only shows up when `--verbose` flag is set.

Reply:
Ok, that sounds good.